config.json

```diff
@@ -149,7 +149,11 @@
       "core": false,
       "difficulty": 4,
       "slug": "sum-of-multiples",
-      "topics": null,
+      "topics": [
+        "loops",
+        "conditionals",
+        "mathematics"
```

Comment:
Great suggestions! :) I would also suggest `integers` and `arrays` since they're used in the [tests](https://github.com/exercism/java/blob/master/exercises/sum-of-multiples/src/test/java/SumOfMultiplesTest.java). Do you agree? :)